sky/skylet/autostop_lib.py

```diff
@@ -59,14 +62,22 @@ def set_autostop(idle_minutes: int, backend: Optional[str], down: bool) -> None:
         set_last_active_time_to_now()
 
 
-def set_is_autostopping() -> None:
-    """Sets the is_autostopping flag to True."""
+def set_autostopping_indicator() -> None:
+    """Sets the boot time of the machine when autostop starts.
+
+    This function should be called when the cluster is started to autostop,
+    and the boot time of the machine will be stored in the configs database
+    as an autostop indicator.
```

Comment:
nit: break paras by blank lines?

Reply:
Actually, they should be in the same paragraph. Fixed. Thanks!